Application/Client/src/Admin.cpp

```diff
@@ -0,0 +1,294 @@
+        if(remove_employee_status == "Employee removed successfully") {
+            std::cout << "Employee " << username << " Removed Successfully" << std::endl;
+        } else {
+            std::cout << "Error Removing Employee: " << remove_employee_status << std::endl;
+        }
+    } catch (const std::exception& e) {
+        std::cerr << "An error occurred while removing employee: " << e.what() << std::endl;
+    } catch (...) {
+        std::cerr << "An unknown error occurred while removing employee" << std::endl;
+    }
+}
+
+void Admin::handleAddMenuItem() {
+    try {
+        std::string menuItemDetails = getMenuItemDetails();
+        clientSocket.sendMessage(static_cast<int>(RequestType::ADD_MENU_ITEM_REQUEST), menuItemDetails);
+        std::string add_menu_item_status = clientSocket.receiveMessage();
+        add_menu_item_status = Utils::splitStringbyTab(add_menu_item_status)[1];
+        if(add_menu_item_status == "Menu item added successfully") {
+            std::cout << "\nMenu item Added Successfully" << std::endl;
+            updateNotificationToAll("New Menu Item Added");
+        } else {
+            std::cout << "Error Adding Menu item: " << add_menu_item_status << std::endl;
+        }
+    } catch (const std::exception& e) {
+        std::cerr << "An error occurred while adding menu item: " << e.what() << std::endl;
+    } catch (...) {
+        std::cerr << "An unknown error occurred while adding menu item" << std::endl;
+    }
+}
+
+void Admin::handleRemoveMenuItem() {
+    try {
+        std::string menu_item;
+        std::cout << "Enter Menu Item Name: ";
+        getline(std::cin, menu_item);
+        
+        clientSocket.sendMessage(static_cast<int>(RequestType::REMOVE_MENU_ITEM_REQUEST), menu_item);
+        std::string remove_menu_item_status = clientSocket.receiveMessage();
+        remove_menu_item_status = Utils::splitStringbyTab(remove_menu_item_status)[1];
+        if(remove_menu_item_status == "Menu item removed successfully") {
+            std::cout << "Menu Item Removed Successfully" << std::endl;
+            updateNotificationToAll("Menu Item Removed: " + menu_item);
+        } else {
+            std::cout << "Error Removing Menu item: " << remove_menu_item_status << std::endl;
+        }
+    } catch (const std::exception& e) {
+        std::cerr << "An error occurred while removing menu item: " << e.what() << std::endl;
+    } catch (...) {
+        std::cerr << "An unknown error occurred while removing menu item" << std::endl;
+    }
+}
+
+std::string Admin::getEmployeeDetails() {
+    std::string user_name = getInput("Enter Username: ");
+    std::string employee_id = getInput("Enter Employee ID: ");
+    std::string first_name = getInput("Enter First Name: ");
+    std::cout << "Enter Last Name: ";
+    std::string last_name;
+    std::cin.ignore();
+    std::getline(std::cin, last_name);
+
+    return user_name + "\t" + first_name + "\t" + last_name + "\t" + employee_id;

```

Comment:
All UI related things can be taken care of separately as discussed in sessions. Please check and update everywhere.